### ISSUE TITLE
The accuracy of each datasets in all sessions withouts session harmony metric

### ISSUE THREAD

**Issue by huangshuai0605** [noorahmedds/OrCo#4]
May you show the direct accuracy results so that people later can compare them more easily？

I've found that many works on arXiv show inconsistent results about you, which may have undermined the accurate representation of your work performance.

**Comment by noorahmedds** [comment]
@huangshuai0605 thanks for noting, I believe most papers report average accuracy which introduce a bias towards the base classes which seems to be the standard convention. However, these numbers _are_ reported on the final page of the supplementary. If there is any confusion with how we compute these numbers, I am happy to help

**Comment by huangshuai0605** [comment]
Thank you, I have found the information I needed in your supplementary materials. I noticed that the performance on CUB-200 is not very good, while the other two datasets, especially miniImageNet, are performing very well. It seems that there is no single method that can perform well on both the FSCIL fine-grained dataset CUB and other coarse-grained datasets at the same time. I'm curious why this is the case and would like to ask for your insights.

**Comment by noorahmedds** [comment]
I agree. Getting good accuracy on the CUB dataset was also quite a challenge. I believe its generally difficult for these low capacity CNNs used for FSCIL which are double as handicapped due to the wide receptive fields and pooling, eventually not entirely capturing the high frequencies in these fine-grained datasets. Which is also why most methods used imagenet weights directly for the base session to compensate for limited base training data. We did try label smoothing and temperature scaling which improved results for across the board. Although, the paper eventually did not contain these experiments. In general, I guess a one fit-all solution doesn't exist currently in FSCIL research.

**Comment by huangshuai0605** [comment]
Thank you for your prompt and helpful response. I wish you fruitful academic achievements.
